modules/serverless-sinkbinding-ossm.adoc

```diff
@@ -0,0 +1,168 @@
+
+* You have integrated {SMProductShortName} with {ServerlessProductName}.
+
+.Procedure
+
+. Create a `Service` in a namespace that is member of the `ServiceMeshMemberRoll`.
++
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+  namespace: <namespace> <1>
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true" <2>
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+    spec:
+      containers:
+      - image: quay.io/openshift-knative/knative-eventing-sources-event-display:latest
+----
+<1> A namespace that is member of the `ServiceMeshMemberRoll`.
+<2> Injects {SMProductShortName} sidecars into the Knative service pods.
+
+. Apply the `Service` resource.
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Create a `SinkBinding`.
++
+[source,yaml]
+----
+apiVersion: sources.knative.dev/v1
+kind: SinkBinding
+metadata:
+  name: bind-heartbeat
+  namespace: <namespace> <1>
+spec:
+  subject:
+    apiVersion: batch/v1
+    kind: Job <2>
+    selector:
+      matchLabels:
+        app: heartbeat-cron
+
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+----
+<1> A namespace that is member of the `ServiceMeshMemberRoll`.
+<2> In this example, any Job with the label `app: heartbeat-cron` will be bound to the event sink.
+
+. Apply the `SinkBinding` resource.
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Create a `CronJob`:
++
+[source,yaml]
+----
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: heartbeat-cron
+  namespace: <namespace> <1>
+spec:
+  # Run every minute
+  schedule: "* * * * *"
+  jobTemplate:
+    metadata:
+      labels:
+        app: heartbeat-cron
+        bindings.knative.dev/include: "true"
+    spec:
+      template:
+        metadata:
+          annotations:
+            sidecar.istio.io/inject: "true" <2>
+            sidecar.istio.io/rewriteAppHTTPProbers: "true"
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: single-heartbeat
+              image: quay.io/openshift-knative/heartbeats:latest
+              args:
+                - --period=1
+              env:
+                - name: ONE_SHOT
+                  value: "true"
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+----
+<1> A namespace that is member of the `ServiceMeshMemberRoll`.
+<2> Injects {SMProductShortName} sidecars into the `CronJob` pods.
+
+. Apply the `CronJob` resource.
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+.Verification
+
+You can verify that the events were sent to the Knative event sink by looking at the message dumper function logs.
```

Comment:
Might be more direct if reworded as: "To verify that the events were sent to the Knative event sink, look at the message dumper function logs."

Reply:
Added your suggestions